src/operations.py

```diff
@@ -198,7 +198,7 @@ def shift_vertically(item, xdata, ydata, left_scale, right_scale, items):
                 new_ydata = [value + shift_value_linear for value in ydata]
             else:
                 new_ydata = [value * 10 ** shift_value_log for value in ydata]
-        return xdata, new_ydata, False, False
+            return xdata, new_ydata, False, False
```

Comment:
```suggestion
            return xdata, new_ydata, False, False
        return xdata, ydata, False, False
```
Oh this was my bad for not testing correctly. It was meant as a failsafe, if for some absurd reason the logic didn't work.
Something like this should then work however

Reply:
Needs one indentation less, which I added in the latest commit.